overview/features.rst

```diff
@@ -0,0 +1,103 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team Edition and Enterprise Edition.
+
+Team Edition Features
+----------------------------------------------
+
+1. **1-1 and group messaging**
+
+.. image:: ../images/messaging.png
+
+2. **File sharing**
+
+.. image:: ../images/file_sharing.png
+
+3. **Custom Emojis**
+
+.. image:: ../images/custom_emojis.png
+
+4. **Webhooks**
+
+.. image:: ../images/webhooks.png
+
+5. **Slash commands**
+
+.. image:: ../images/slash_commands.png
+
+6. **Plug-in framework**
+
+.. image:: ../images/plugin_framework.png
+
+7. **Mobile applications**
+
+.. image:: ../images/mobile_apps.png
+
+8. **Desktop applications**
+
+.. image:: ../images/desktop_apps.png
+
+9. **Multi-team support**
+
+.. image:: ../images/multi-team_support.png
+
+10. **Threaded conversations**
+
+.. image:: ../images/threaded_conversations.png
+
+11. **Theme colors**
+
+.. image:: ../images/theme_colors.png
+
+12. **Multi-language support**
+
+.. image:: ../images/multi-language_support.png
+
+Enterprise Edition Features
+----------------------------------------------
+
+1. **AD/LDAP SSO**
+
+.. image:: ../images/AD_LDAP_SSO.png
+
+2. **Advanced Permissions**
+
+.. image:: ../images/advanced_permissions.png
+
+3. **MFA**
+
+.. image:: ../images/MFA.png
+
+4. **Enterprise support**
+
+.. image:: ../images/enterprise_edition_support.png
+
+5. **SAML-based SSO**
+
+.. image:: ../images/SAML-based_SSO.png
```

Comment:
file name here needs to be fixed
<img width="361" alt="screenshot 2017-12-15 17 50 05" src="https://user-images.githubusercontent.com/9354841/34063604-15937e2e-e1c1-11e7-99a0-22d141fec4bb.png">